docs/user-manuals/sidecarset.md

```diff
@@ -216,6 +216,42 @@ spec:
 ```
 **Note**: Users must ensure that the corresponding Secrets have already existed in the namespaces where Pods need to pull the private images. Otherwise, pulling private images will not succeed.
 
+### version control for injection
+**FEATURE STATE:** Kruise v1.3.0
+
+SidecarSet will record historical revision of some fields such as `containers`, `volumes`, `initContainers`, `imagePullSecrets` and `patchPodMetadata` via ControllerRevision. You can easily select a specific historical revision to inject when creating Pods, rather than always inject the latest revision of sidecar.
+
+Based on this feature, users can address the risk of SidecarSet publishing due to scaling and rolling workload.
+
+**Note: SidecarSet records ControllerRevision in the same namespace as Kruse Manager.**
+
+#### select revision via ControllerRevision name
+```yaml
+apiVersion: apps.kruise.io/v1alpha1
+kind: SidecarSet
+metadata:
+  name: sidecarset
+spec:
+  ... ...
+  injectionStrategy:
+    revisionName: <specific-controllerrevision-name>
```

Comment:
Is there a simple example that tells the user how to find the sidecarSet controllerRevision?